components/multiselect/multiselect_list.jsx

```diff
@@ -118,9 +128,17 @@ export default class MultiSelectList extends React.Component {
         );
     }
 
+    handleScroll = () => {
+        const scrolledTill = this.listScrollBar.current.getValues().scrollTop;
+        if (scrolledTill > 1000 && !this.props.loading) {
```

Comment:
If we use scrollTop then we will be loading new members whenever we hit 1000 from top i guess you meant to calculate from 1000 from bottom. Rather than 1000px we should be loading based on percentage such as 30% from the bottom so we would not hit issues with different screen resolutions.

Reply:
Yes, that sounds good. But the scrollbars API is currently giving in number, will work on that how to do with percentages.